src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -85,7 +65,10 @@
 public class ExtensionsRunner {
 
     private static final Logger logger = LogManager.getLogger(ExtensionsRunner.class);
-    private static final String NODE_NAME_SETTING = "node.name";
+    /**
+     * This String get a call from {@link GetNetty4Transport}.
+     */
+    public static final String NODE_NAME_SETTING = "node.name";
```

Comment:
You can move this to the newly created Netty4Transportclass and make it private.